### PR TITLE
test VS editor

### PR DESCRIPTION
**Pull request by mindspin311**
test

### CODE REVIEW

**Review by chenfeiyu132 — APPROVED** [review]

**Review by chenfeiyu132 — APPROVED** [review]

**Review by chenfeiyu132 — APPROVED** [review]

File: slack-java-client-examples/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Welcome to our examples! You can see how we configure the client in a few different ways in the example module, and you can find specific examples of using the client in the `examples` package. Feel free to add new ones, or let us know if these are misleading or insufficient. 
+
+I am a test of VS
```
Comment (jbreckman) [inline slack-java-client-examples/README.md:3]: not good

File: slack-java-client-examples/README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 Welcome to our examples! You can see how we configure the client in a few different ways in the example module, and you can find specific examples of using the client in the `examples` package. Feel free to add new ones, or let us know if these are misleading or insufficient. 
+
```
Comment (jbreckman) [inline slack-java-client-examples/README.md:2]: staged comment

**Review by jbreckman — COMMENTED** [review]
i'm not a fan

File: slack-java-client-examples/README.md [hunk 3]
```diff
@@ -1 +1,3 @@
 Welcome to our examples! You can see how we configure the client in a few different ways in the example module, and you can find specific examples of using the client in the `examples` package. Feel free to add new ones, or let us know if these are misleading or insufficient.
```
Comment (jbreckman) [inline slack-java-client-examples/README.md:1]: third comment

**Review by jbreckman — COMMENTED** [review]
nice!

File: slack-java-client-examples/README.md (hunk 1 above)
Comment (mindspin311) [reply slack-java-client-examples/README.md:3]: I blame VS Code for doing this

**Comment by mindspin311** [comment]
Feiyu contributing to our public repo now 🥳

**Pull request merged**